NSS-Likeable-Lemurs/WorkforceManagement
# ticket#3 done

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/WorkforceManagement/issues/3)

## Description of Proposed Changes
- Created model for Employee
- Created model for department
- Created mvc for Employee

## Steps to Test

Outline the steps to test

```
1.Insert Employee and department data into your SQL
2.In the visual studio - do the integration testing for FirstName,LastName and Department and make sure it pass the test.
3.Run the BangazoneWorkForceMVC
4.Click Employee in the Nav Bar
5.Check the list of employees there.
```
## Impacted Areas in Application

List general components of the application that this PR will affect:

* EmployeeTest.cs   * Employee-Index.cshtml

## Mentions @username

@streatorward 
@HelenChalmers 
@Akeen11 
@kellycook301 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
